### PR TITLE
Update default log level from debug to info

### PR DESCRIPTION
**Pull request by jackfranklin**
@guybedford I missed this in the original PR :cry:

### CODE REVIEW

File: lib/ui.js [hunk 1]
```diff
@@ -95,7 +96,7 @@ exports.log = function(type, msg) {
     msg = format[type](msg);
 
   if (logging) {
-    if ((logLevel === 'debug') ||
+    if ((logLevel === DEFAULT_LOG_LEVEL) ||
```
Comment (jackfranklin) [inline lib/ui.js]: this is so any messages without a `type` get shown only when the log level is at the default

**Comment by guybedford** [comment]
Ah, caught this in https://github.com/jspm/jspm-cli/commit/fa046c3e2ee6de5ac88298b641a68bf13ae447f7, so we should be ok?

**Comment by jackfranklin** [comment]
I think that will mean that any message that doesn't have a type will now never be logged? I think you still need to have that conditional that just logs absolutely anything if the log level is `info`.

**Comment by guybedford** [comment]
@jackfranklin https://github.com/jspm/jspm-cli/commit/bc5ad99c2095e40c03bfae23c33f5f51ac2d91fb? Let me know if I'm missing something here though.

**Comment by jackfranklin** [comment]
Ah yeah, I think that covers it :+1:

**Comment by guybedford** [comment]
Great, thanks!

**Pull request closed without merging**